games-puzzle/pathological/pathological-1.1.3.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="An enriched clone of the game Logical"
+DESCRIPTION="Pathological is an enriched clone of the game \
+"Logical" by Rainbow Arts. \
+To solve a level, fill each wheel with four marbles of matching color. \
+Various board elements such as teleporters, switches, filters, etc., \
+make the game interesting and challenging. \
+New levels can be created using your favorite text editor.clea"
+HOMEPAGE="http://pathological.sourceforge.net/"
+COPYRIGHT="2003 John-Paul Gignac"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/pathological/files/pathological/$portVersion/pathological-$portVersion.tar.gz"
+CHECKSUM_SHA256="813ef0812ba6b3218b35c2ddea423c9c6ee912925824497f52d5310bc76f85de"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	pathological = $portVersion
+	"
+REQUIRES="
+        haiku
+        "
+
+BUILD_REQUIRES="
+        haiku_devel
+        "
+
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs
```

Comment:
You could use DESTDIR here to set a path to $appsDir/Pathological, that way it should be ok to build, with one addition ...